### DISCUSSION TITLE
Display formatting bar when using markdown

### DISCUSSION THREAD

**Discussion by A248** [NamelessMC/Nameless#2141] [Ideas]
**Please describe the feature in as much detail as possible**

When `Configuration > Post Formatting Type` is set to _HTML_, users have access to a formatting bar when creating a forum topic.
However, when using **Markdown**, the formatting bar is not displayed. Not all users are familiar with markdown, so it would be helpful to retain some sort of formatting bar despite using markdown. Such a formatting bar would also help users learn the markdown style.

**Using HTML**
![Screenshot 2020-02-08 at 3 48 33 PM](https://user-images.githubusercontent.com/22414680/74091837-85dcbe00-4a8a-11ea-9ae0-d565ff8b03ac.png)
**Using Markdown**
![Screenshot 2020-02-08 at 3 51 50 PM](https://user-images.githubusercontent.com/22414680/74091889-f1269000-4a8a-11ea-94bf-cf6649afc67d.png)

**Comment by samerton** [comment]
This was an intentional design decision when I implemented the Markdown editor, to be honest it will probably be removed for the full v2 release as it doubles the maintenance whenever features are added to forum posts.

**Comment by Derkades** [comment]
Aren't there markdown editors that also allow you to use HTML? Like here on GitHub

**Comment by A248** [comment]
^ That's what I figured. Also, I hope the Markdown editor doesn't go. I like Markdown...
